documentation/staging/content/userguide/managing-operators/preparation.md

```diff
@@ -437,3 +355,82 @@ the operator REST API,
 setting operator pod labels,
 setting operator pod annotations,
 and Istio.
+
+### Special use cases
+
+If applicable, please review the following special use cases.
+
+#### How to download the Helm chart if Internet access is not available
+
+At a high-level, you use `helm pull` to download a released version of the Helm chart and move it to the machine with no Internet access,
+so that then you can run `helm install` to install the operator.  
+
+The steps are:
+1. On a machine with Internet access, to download the chart to the current directory, run `$ helm pull weblogic-operator --repo https://oracle.github.io/weblogic-kubernetes-operator/charts --destination .`
```

Comment:
This step should mention the version option and how to get the list of versions...

Reply:
(You mentioned it earlier - but it's potentially a necessary part of this particular step...)

Reply:
@tbarnes-us I also mention it after the steps but perhaps, I should mention it here instead.

Reply:
Or in both places - the version is needed for the "repo" case too.  Especially in production, where control over the version is usually fairly important...

Reply:
@tbarnes-us I **did** put the version info in that section earlier in the doc.